flake.nix

```diff
@@ -27,42 +31,54 @@
         # Adds all contracts in current release to all chains across networks
         zero-nix.flakeModules.valence-contracts
       ];
-      perSystem = { lib, config, inputs', ... }:
-        let zeroNixPkgs = inputs'.zero-nix.packages; in
-          {
-            valence-contracts.upload = true;
-            upload-contracts = {
-              network-defaults = { name, ... }: {
-                data-dir = "./${name}/contracts-data";
-                program-manager-chains-toml = ./${name}/chains.toml;
-                chain-defaults = {
-                  contract-defaults = {
-                    # Only use valence-contracts-main for contracts in list above
-                    package = lib.mkDefault zeroNixPkgs.valence-contracts-v0_1_2;
-                  };
-                  contracts = {
-                    valence_drop_liquid_staker.package = zeroNixPkgs.valence-contracts-main;
-                    valence_drop_liquid_unstaker.package = zeroNixPkgs.valence-contracts-main;
-                  };
-                };
+      perSystem = {
+        lib,
+        inputs',
+        ...
+      }: let
+        zeroNixPkgs = inputs'.zero-nix.packages;
+      in {
+        valence-contracts.upload = true;
+        upload-contracts = {
+          network-defaults = {name, ...}: {
+            data-dir = "./${name}/contracts-data";
+            program-manager-chains-toml = ./${name}/chains.toml;
+            chain-defaults = {
+              contract-defaults = {
+                # Use contracts from release by default
+                package = lib.mkDefault zeroNixPkgs.valence-contracts-v0_1_2;
               };
-              networks.mainnet.chains = {
-                neutron = {
-                  max-fees = "1000000";
-                };
-                juno = {
-                  max-fees = "1000000";
-                };
-                terra = {
-                  max-fees = "6000000";
-                };
+            };
+          };
+          networks.mainnet.chains = {
+            neutron = {
+              max-fees = "1000000";
+              contracts = {
+                valence_drop_liquid_staker.package = zeroNixPkgs.valence-contracts-main;
+                valence_drop_liquid_unstaker.package = zeroNixPkgs.valence-contracts-main;
               };
-              networks.testnet.chains = {
-                neutron = {
-                  max-fees = "1000000";
-                };
+            };
+            juno = {
+              max-fees = "1000000";
+            };
+            terra = {
+              max-fees = "6000000";
+            };
+            osmosis = {
+              max-fees = "1000000";
+              contracts = {
+                valence_astroport_lper.enable = false;
+                valence_astroport_withdrawer.enable = false;
```

Comment:
the enable option can be used now to disable specific contracts for new chains
full list of contracts included by default can be found here: https://github.com/timewave-computer/zero.nix/blob/4335f1f0d43e0d4c5d12ab4065fef1ad10dcda7e/flakeModules/valence-contracts.nix#L99

But in general zero.nix uploads all contracts in the latest release for chains which it makes sense to add them